routes/furniture.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const { pool, db } = require('../server/db');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'db' is assigned a value but never used.

Reply:
Because of the way we set up these SQL queries, let's change all instances of `pool` to use `db` instead. More info as to why will come, but essentially, `db` gives us more flexibility to add attributes conditionally.

Reply:
For future reviews, also remember to remove any unused values or variables.